src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const elements = [...document.querySelectorAll('span.population')];
+
+const populationArr = elements.map((element) => {
+  return Number(element.innerText.split(',').join(''));
+});
+
+const totalPopulation = populationArr.reduce((acc, oneCountryPopulation) => {
+  return acc + oneCountryPopulation;
+}, 0);
```

Comment:
You are using the `reduce` method to calculate the total population, which is a great use of this method. However, remember that the `reduce` method can be a bit difficult to understand for people who are not familiar with it. A comment explaining what this code does could be helpful for other developers reading your code.